src/core_plugins/table_vis/public/table_vis.js

```diff
@@ -59,7 +59,7 @@ function TableVisTypeProvider(Private) {
       defaults: {
         perPage: 10,
         showPartialRows: false,
-        showMeticsAtAllLevels: false,
```

Comment:
@cjcenizal We use these variables in our dashboards. I'm wondering now if this is a breaking change or the old dashboards using `showMeticsAtAllLevels` will continue to work?

Reply:
@elastic/kibana-visualizations Could you please take a look at this and let me know if we need to revert this or perhaps add support for the misspelled parameter to ensure BWC?

Reply:
we need to add some code to ensure bwc where not just internal variables were changed but properties that are stored in the saved objects.

for visualizations this should be easy, take a look at `ui/vis/vis_update_state`

i'll go over this PR tomorrow and see what needs to be updated on visualize side

Reply:
@ppisljar Thanks, I've set up a meeting to discuss this.